src/main/java/duke/MainWindow.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+/**
```

Comment:
Perhaps you should leave a line between your import and your Javadoc comment?

Reply:
Alright thanks